VirtoCommerce.Storefront/appsettings.json

```diff
@@ -41,6 +40,9 @@
       // Example values: 'default_theme' will map to this path  'wwwroot/cms-content/default_theme'
       "BaseThemeName": ""
     },
+    "AppSettings": {
+      "PlatformUrlHost": "http://localhost/"
```

Comment:
Why do we need this?

Reply:
I use it in [the preview template](https://github.com/VirtoCommerce/vc-content/blob/7766bdd846b0bab00f309a6c2d42db8a52507555/theme/templates/json-preview.liquid#L5). this is the single way that i found to say preview script about platfrom and page builder.

Reply:
But we already have this URL: VirtoCommerce.Endpoint

Reply:
i haven't access to this setting in liquid template. or have, but don't know how to get it. explain me if you know.

Reply:
@artem-dudarev  do you have any suggestions about this question?

Reply:
If `VirtoCommerce.Endpoint` is not available in Liquid templates you should make it available instead of adding duplicate settings.

Reply:
ok, i removed it from settings. I've found the other way for this.